.gitignore

```diff
@@ -1,3 +1,6 @@
+# DataBase
+/data/
```

Comment:
The concrete path is `./data/`

Reply:
Nope, my path appears to be correct.<br>
![Screenshot_1](https://user-images.githubusercontent.com/54960216/109382166-d3bfa080-78ef-11eb-9b22-4938d7cb27ef.png)
![Screenshot_3](https://user-images.githubusercontent.com/54960216/109382170-d7ebbe00-78ef-11eb-92de-8a9427868493.png)